L1Trigger/Configuration/python/L1Trigger_EventContent_cff.py

```diff
@@ -93,3 +94,20 @@
         'keep LumiSummary_lumiProducer_*_*')
 )
 
+
+if eras.stage2L1Trigger.isChosen():
```

Comment:
I took it from another L1Trigger cff file, assuming it was the right thing to do.
What's the right syntax?